code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -100,4 +142,74 @@
 		for (var/datum/role/R in members)
 			var/mob/M = R.antag.current
 			to_chat(M, "<span class='sinister'>This number might rise up to 9 as more people arrive aboard the station.</span>")
+	AnnounceObjectives()
 	..()
+
+
+/datum/faction/bloodcult/proc/GetDepartmentName(var/area/D)
```

Comment:
This proc seems unused and the implementation seems wrong. The switch only checks if the types are equal, subtypes are not considered equals. I think.
Remove this if it's unused, fix it and use it if it's meant to be used.